proarc-common/src/main/java/cz/cas/lib/proarc/common/fedora/RemoteStorage.java

```diff
@@ -319,6 +319,18 @@ public void purge(String logMessage) throws DigitalObjectException {
             }
         }
 
+        public void purgeDatastream(String datastream ,String logMessage) throws DigitalObjectException {
+            try {
+                FedoraClient.purgeDatastream(getPid(), datastream).logMessage(qpEncode(logMessage)).execute(client);
+            } catch (FedoraClientException ex) {
+                if (ex.getStatus() == Status.NOT_FOUND.getStatusCode()) {
+                    throw new DigitalObjectNotFoundException(getPid(), ex);
```

Comment:
Minimálně by bylo dobré to dopsat do zdrojáku, aby se to nemuselo znovu analyzovat. Jinak přidání testu není složité viz. test package `cz.cas.lib.proarc.common.fedora`. Jen je potřeba nastavit přístup k běžící Fedoře viz. `proarc-common/pom.xml` a templeta `junit.properties`.